utils/ffmpegvideodecode/ffmpeg_video_dec.cpp

```diff
@@ -452,9 +461,29 @@ int FFMpegVideoDecoder::HandlePictureDecode(RocdecPicParams *pPicParams) {
 
     if (no_multithreading_) {
         DecodeAvFrame(av_pkt, dec_frames_[av_frame_cnt_]);
+        int num_frames_to_display = decoded_pic_cnt_;
+        while (num_frames_to_display) {
+            RocdecParserDispInfo dispInfo = {0}; // dispinfo is not used in ffmpeg decoder, so setting it to zero
+            HandlePictureDisplay(&dispInfo);
+            num_frames_to_display--;
+        };
+        if (!last_packet_.payload_size && !end_of_stream_) {
+            AVPacket pkt = { 0 };
+            DecodeAvFrame(&pkt, dec_frames_[av_frame_cnt_]);
+            int num_frames_to_display = decoded_pic_cnt_;
+            while (num_frames_to_display) {
+                RocdecParserDispInfo dispInfo = { 0 }; // don't care about this as this will be igonored
```

Comment:
Please modify the comment for this line to be similar to line 466. Also change `{ 0 }` -> `{0}`